### PR TITLE
[ADD] New module mail_sent.

### PR DESCRIPTION
**Pull request by yajo**
Adds a view where the user can see the sent mails.

cc @rafaelbn

### CODE REVIEW

File: mail_sent/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+To use this module, you need to:
+
+* Go to *Messaging > Messaging > Sent*.
+* If you wish to see only unread ones, click on the counter bubble (if any).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/205/8.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (pedrobaeza) [inline mail_sent/README.rst]: Remove this

File: mail_sent/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+.. branch is "8.0" for example
+
+For further information, please visit:
```
Comment (pedrobaeza) [inline mail_sent/README.rst]: And this also

File: mail_sent/README.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/ social/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and
+welcomed feedback `here <https://github.com/OCA/
+social/issues/new?body=module:%20 mail_sent%0Aversion:%20
+8.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline mail_sent/README.rst]: Put Odoo version, not module version

**Comment by pedrobaeza** [comment]
You can put an icon like https://openclipart.org/detail/110941/folder-sent-mail or https://openclipart.org/detail/197922/mono-mail-send

**Comment by rafaelbn** [comment]
Tested in runbot :+1:

**Comment by sergio-teruel** [comment]
Tested in runbot :+1:

**Comment by dreispt** [comment]
Great addition! 
Other than Pedro's comments, code review :+1:

**Comment by yajo** [comment]
All fixed.

**Comment by rafaelbn** [comment]
I think this is ready to merge, isn't it?

**Comment by pedrobaeza** [comment]
There's a little comment from my part, but nothing critical, so I merge.

**Comment by yajo** [comment]
@pedrobaeza That section should be in the template then.

**Comment by pedrobaeza** [comment]
Yeah, it can be an improve. Can you make the PR?

**Pull request merged**